inspirehep/modules/authors/receivers.py

```diff
@@ -17,18 +17,20 @@
 # along with INSPIRE; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
-import numpy as np
 import uuid
 
 from beard.clustering import block_phonetic
+
 from invenio_records.signals import (
     before_record_index,
     before_record_insert,
 )
 
+import numpy as np
+
 
 @before_record_index.connect
-def assign_signature_block(recid, json):
+def assign_signature_block(recid, json, index):
```

Comment:
It's because the signature of the signal has changed. The `index` argument here is just dummy.

Maybe could be substitute with `*args, **kwargs`?